CleverPush/Source/CleverPushInstance.m

```diff
@@ -1964,6 +1974,7 @@ - (void)startLiveActivity:(NSString*)activityId pushToken:(NSString*)token onSuc
                                  activityId, @"iosLiveActivityId",
                                  token, @"iosLiveActivityToken",
                                  subscriptionId, @"subscriptionId",
+                                 [NSNumber numberWithBool:autoRequestNotificationPermission], @"hasNotificationPermission",
```

Comment:
Here we do not want to send the value of `autoRequest`. Instead we want to send the authorizationStatus - has user given the permission?